pkg/nsxt/config/config_ini_legacy.go

```diff
@@ -28,6 +28,10 @@ func (cfg *NsxtConfigINI) ValidateConfig() error {
 		if cfg.Password == "" {
 			return errors.New("password is empty")
 		}
+	} else if cfg.ClientAuthKeyFile != "" {
+		if cfg.ClientAuthCertFile == "" {
+			return errors.New("client cert file is empty")
```

Comment:
"client cert file is required if client key file is provided" or something like that

Reply:
+1